src/crewai/knowledge/knowledge.py

```diff
@@ -0,0 +1,53 @@
+from typing import List
+
+from pydantic import BaseModel, ConfigDict, Field
+
+from crewai.knowledge.embedder.base_embedder import BaseEmbedder
+from crewai.knowledge.embedder.fastembed import FastEmbed
+from crewai.knowledge.source.base_knowledge_source import BaseKnowledgeSource
+
+
+class Knowledge(BaseModel):
```

Comment:
Lets write some docs about:
1. how to use this
2. setting your own custom embedder for this